src/SFML.Graphics/RenderTexture.cs

```diff
@@ -525,8 +525,7 @@ protected override void Destroy(bool disposing)
         private Texture myTexture = null;
 
         #region Imports
-        [Obsolete("sfRenderTexture_create is obselete. Use sfRenderTexture_createWithSettings instead.")]
-        [DllImport(CSFML.graphics, CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity]
+        [DllImport(CSFML.graphics, CallingConvention = CallingConvention.Cdecl), SuppressUnmanagedCodeSecurity, Obsolete]

```

Comment:
Doesn't this remove the hint for `Use sfRenderTexture_createWithSettings instead.`?

Reply:
It does remove that hint, but only for calling `sfRenderTexture_create()` directly.  Since that DllImport is private, then end user would never see that warning anyways. The warning still exists on the public-facing API.